<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SudokuDialog</class>
 <widget class="QDialog" name="SudokuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>10</width>
     <height>800</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ReStart">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>200</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewGame">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>190</y>
     <width>200</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>新的游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Answer">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>370</y>
     <width>200</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>查看答案</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnClose">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>610</y>
     <width>200</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OutPut">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>490</y>
     <width>200</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>输出cnf文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HolesEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>280</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnClose</sender>
   <signal>clicked()</signal>
   <receiver>SudokuDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
